### ISSUE TITLE
Undefined name 'model_urls' in ./models/resnet_sn.py

### ISSUE THREAD

**Issue by cclauss** [switchablenorms/Switchable-Normalization#2]
Each _undefined name_ has the potential to raise NameError at runtime.

flake8 testing of https://github.com/switchablenorms/Switchable-Normalization on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./models/resnet_sn.py:159:50: F821 undefined name 'model_urls'
        model.load_state_dict(model_zoo.load_url(model_urls['resnet18']))
                                                 ^
./models/resnet_sn.py:171:50: F821 undefined name 'model_urls'
        model.load_state_dict(model_zoo.load_url(model_urls['resnet34']))
                                                 ^
./models/resnet_sn.py:183:50: F821 undefined name 'model_urls'
        model.load_state_dict(model_zoo.load_url(model_urls['resnet50']))
                                                 ^
./models/resnet_sn.py:195:50: F821 undefined name 'model_urls'
        model.load_state_dict(model_zoo.load_url(model_urls['resnet101']))
                                                 ^
./models/resnet_sn.py:207:50: F821 undefined name 'model_urls'
        model.load_state_dict(model_zoo.load_url(model_urls['resnet152']))
                                                 ^
5     F821 undefined name 'model_urls'
5
```

**Comment by JiaminRen** [comment]
Currently, we release the model of ResNet50v1+SN , you can find the pretrained model download link in Model Zoo of README.md, the model_urls in resnet_sn.py is unavailable.

**Comment by pluo911** [comment]
SN is easy to use with the other networks. Will release more examples.
